packages/gallery-plugin/src/GalleryComponent.ts

```diff
@@ -160,8 +160,13 @@ export class GalleryComponent extends AbstractComponent {
         currentActive?.classList.remove(ACTIVE_CLASS);
 
         if (id) {
-            const nextActive = this.items.querySelector(`[data-${GALLERY_ITEM_DATA_KEY}="${id}"]`);
-            nextActive?.classList.add(ACTIVE_CLASS);
+            const nextActive = this.items.querySelector(`[data-${GALLERY_ITEM_DATA_KEY}="${id}"]`) as HTMLElement;
+            if (nextActive) {
+                nextActive.classList.add(ACTIVE_CLASS);
+                // nextActive.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'start' });
+                this.items.parentElement.scrollLeft =
+                    nextActive.offsetLeft + nextActive.clientWidth / 2 - this.items.parentElement.clientWidth / 2;
```

Comment:
```suggestion
                this.items.scrollLeft = nextActive.offsetLeft + nextActive.clientWidth / 2 - this.items.clientWidth / 2;
```

the scrolling element changed in this commit https://github.com/mistic100/Photo-Sphere-Viewer/commit/69380e4709aa63abfc408da05c37f664563c8a89 (version 5.4.3)